packages/driver/src/cypress/runner.ts

```diff
@@ -492,7 +492,57 @@ const hasOnly = (suite) => {
   )
 }
 
-const normalizeAll = (suite, initialTests = {}, setTestsById, setTests, getRunnableId, getHookId, getOnlyTestId, getOnlySuiteId, createEmptyOnlyTest) => {
+const pruneSuite = (emptySuite) => {
```

Comment:
Is the video snippet up-to-date with the latest code? I was noticing the suite title was showing in the Command Log with no associated tests. I would not expect an empty suite to be displayed.

Reply:
No, that video snippet is older and from an initial implementation. It should now filter out the empty suites.